### DISCUSSION TITLE
Ability to configure multiple values for org.owasp.csrfguard.domainOrigin

### DISCUSSION THREAD

**Discussion by bpapez** [OWASP/www-project-csrfguard#169] [General]
We have a requirement for being able to configure multiple values for the `org.owasp.csrfguard.domainOrigin` property. It could either be a regular expression, like:

`org.owasp.csrfguard.domainOrigin = example.(ch|swiss)`

or a list like:
`org.owasp.csrfguard.domainOrigin = example.ch,example.swiss`

As far as I can see this value is only used as parameter in the `isValidDomain` function of `csrfguard.js` . Should we provide a PR to handle multiple values in the original `csrfguard.js` or do you think this should just be solved in custom adaptations? Which solution do you prefer, the regexp or the list?

Regards,
Benjamin

**Comment by forgedhallpass** [comment]
Hello @bpapez,

I assume generally you wouldn't have too many domains to configure. If there are only a few, then I'd prefer the list approach, because it's less error prone.

Feel free to provide a PR.
Thank you.

**Comment by gflores-jahia** [comment]
Created PR https://github.com/OWASP/www-project-csrfguard/pull/200 for this enhancement.

**Comment by forgedhallpass** [comment]
Released under version [4.3.0](https://github.com/OWASP/www-project-csrfguard/releases/tag/4.3.0)
